examples/producer.go

```diff
@@ -13,9 +13,8 @@ func main() {
 		os.Exit(1)
 	}
 	defer conn.Close()
-
 	p, err := conn.CreateProducer("<station-name>", "<producer-name>")
-	err = p.Produce([]byte("You have a message!"))
+	err = p.Produce([]byte("You have a message!"), map[string][]string{"<key>": {"<value>"}, "<key1>": {"<value1>"}})
```

Comment:
I don't like it this way, this function gets as a second parameter the options, headers should be a part of these options, and not like this but as a dedicated struct of headers